2016-08/august-05.md

```diff
@@ -0,0 +1,73 @@
+
+* We are still on the hunt for the `webpack.io` domain name. We see its registered to a gentleman in China, and would love any Mandarin speakers to help us draft an email to reach out to him for domain purchase as the first few emails have been unsuccessful!! [Marco Pai](https://github.com/MarcoPai) has some slight activity on github. So we aren't giving up hope just yet.
+
+* Currently the `webpack/webpack.io` is a statically generated site, so Juho is going to be working to add React and JS for Dynamic usage in other portions of the page. 
+
+* Documentation Structure has been finalized!!!! Priority number one is now creating the content and merging it into develop! The styling for the page is still a work in progress, however that is our last priority before we "ship the docs". 
+
+* For future we may have Johannes with the help of other UI Designers to create personas that help us and contributors create appropriate language for documentation. Agreed that it was a very-nice-to-have but also a great addition if we ever had time. 
+
+* Would love to have folks work on a SchemaStore schema for the webpack configuration. This we could use to autogenerate some documentation like the API: Configuration page. Also could provide IDE support for people almost out of the box. Need to look into how to do this when we tackle that documentation. 
+
+* It was brought up that we need to create a feasible action plan that we can share with the Dan Abramov's, Kent Dodds', and Patrick Stapleton's of the JS community to pass along and promote. It is in everyones best interest to work together to create documentation and having a strategy to accomplish this is vital. We spent time listing out the immediate MVP documents that would the team agreed could allow us to life the webpack 2.x beta status. 
+
+* Juho created a [Github Milestone for all these tasks that we can give to people](https://github.com/webpack/webpack.io/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Webpack+2+-+Documentation+MVP%22)!!!! Barring any major or breaking changes from popular community loaders or plugins (as well as the core loaders and plugins), once this is finished, we believe this is our 'go-ahead' to lift the beta status. We all agreed that although this wouldn't contain all of the documentation we aspired to tackle, that this would give the (first time) users _that need it the most_, the opportunity to have a _far more_ augmented learning experience than what is available. 
+
+* Everyone also agreed that we could technically accomplish this by the end of August. **Yes people, it's getting more than real, this is an estimated timeline of webpack 2.x being out of beta!** :heart_eyes: :heart_eyes: :heart_eyes:. If all goes well and we meet our deadline, then we can lay out our next milestone for more advanced "revisions" of our existing documentation migrated over to the `webpack/webpack.io` repo.
```

Comment:
I have written a post to help you locate this guy.
[http://gold.xitu.io/post/57a69cd8a633bd00604ade55](http://gold.xitu.io/post/57a69cd8a633bd00604ade55)